app/views/users/show.html.erb

```diff
@@ -1,9 +1,38 @@
-<section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
-  <article class="timeline">
+<div class="col-sm-7">
+  <h3><%= "#{@user.name}" %></h3>
+	  
+	 <p><strong>Friends:</strong></p>
+
+	  <ul class='list-unstyled row g-3'>
+	    <% @user.friends.each do |friend| %>
+	      <li class="col-sm-7 mt-1">
+	        <%= friend.name %>
+	        <% if @user == current_user %>
+	          <%= link_to 'Unfriend', friendships_destroy_path(friendship_id: Friendship.find_request(@user.id, friend.id)), class: 'btn btn-danger btn-sm'  %>
+                <% end %>
+              </li><br>
+	    <% end %>
+	  </ul>
+
+	  
+	  <p><strong>Friend Requests:</strong></p>
+
+	  <ul>
+	    <% @user.pending_friendships.each do |friendship| %>
+	      <li>
+	        <%= User.find(friendship.user_id).name %>
+	        <%= link_to 'Accept', friendships_update_path(friendship_id: friendship.id),  class: 'btn btn-success btn-sm'  %>
+	        <%= link_to 'Reject', friendships_destroy_path(friendship_id: friendship.id),  class: 'btn btn-danger btn-sm' %>
```

Comment:
**Optional**. These links are visible only in the user's profile. It's not very user-friendly because you have to click All users and then the profile. It would be better to give immediate access to the current user profile with a link in the navbar.

You are not showing any messages of success or failure when accepting a friend or deleting one.